packages/ui/src/modal/TokenPrimitive.tsx

```diff
@@ -1,5 +1,5 @@
 import { styled } from '../../stitches.config'
```

Comment:
I don't think we want to make this change in the token primitive, the idea was just to change the total at the bottom of the modal where it says, "You Pay"